tools/travis-build

```diff
@@ -32,7 +36,7 @@ export P4C_RUNTIME_DEPS="cpp \
                      libgc1c2 \
                      libgmp10 \
                      libgmpxx4ldbl \
-                     python"
```

Comment:
should python (python2) be kept around here for the sake of "not breaking downstream projects"?

Reply:
python is installed in line 20.

Reply:
I was just curious because we repeat "python3" but not "python". But I see now, it doesn't matter where it is listed, everything is installed in the same invocation of apt-get anyway.

Reply:
Yes, not sure why we repeat it. There is a "point" with needed python at runtime as well. I don't think it matters.